docs/user-guide/server-side-install-overview.md

```diff
@@ -1,19 +1,122 @@
+</details>
 
-Roles & Responsibilities
+*  **Zowe API Mediation Layer (API ML)**  
+Zowe API Mediation Layer provides a reverse proxy and enables REST APIs by providing a single point of access for mainframe service REST APIs.<details><summary>Click here for more information about API Mediation Layer.</summary> Such APIs include MVS Data Sets, JES, as well as working with z/OSMF. The API ML has dynamic discovery capability for these services and Gateway is also responsible for generating the authentication token used to provide single sign-on (SSO) functionality.</details>
 
-Zowe Key Terms & Concepts: Zowe runtime, Zowe Application Framework, Zowe API Mediation Layer, Z Secure Services (ZSS), Zowe Cross Memory Server (@Joe)
+* **Zowe System Services (ZSS)**  <!-- Is ZSS installed with Zowe runtime or is is part of ZIS installation in the configuring section? -->
+Working closely with ZIS, ZSS serves as one of the primary, authenticated back-ends that communicates with z/OS and provides Zowe with a number of APIs including z/OS Unix files and data sets, control of the plug-ins and services lifecycle, and security management.<details><summary>Click here for more information about Zowe System Services.</summary> The Zowe Desktop delegates a number of its services to ZSS which it accesses through the default http port `7557`. ZSS is written in C and uses native calls to z/OS to provide its services.</details>
 
-Technology Needs: YAML, node.js, USS background details (existing content)
+## Installation of Zowe runtime
+
+Zowe runtime refers to the full, unarchived set of binaries, executable files, scripts, and other elements that are run when Zowe is started.
+Runtime installation includes the following components:  
+* Zowe Application Framework
+* Zowe API Mediation Layer
+* Z Secure Services (ZSS)
+
+<!-- Where does this next paragraph belong? -->
+The Zowe server components also includes a web desktop that runs a number of applications including API Mediation Layer that includes the Single Sign-on (SSO) capability, organization of multiple Zowe servers under a single website, and other useful features for z/OS developers.
+
+**Important:** To run all of these server-side components, or to run Zowe Application Framework, installation of the Zowe Cross Memory Server (ZIS) is required. The Zowe cross memory server (ZIS) provides privileged cross-memory services to the Zowe Desktop and runs as an APF-authorized program.<details><summary> Click here for more information about ZIS.</summary>The same cross memory server can be used by multiple Zowe desktops. The cross memory server is needed to be able to log on to the Zowe desktop and operate its apps such as the Code Editor.</details>
```

Comment:
Here ZIS is detailed but ZIS is used before, so maybe better to introduce it earlier

Reply:
Also i believe APIML does not use Cross memory server (ZIS) so it is required only for App Framework

Reply:
SO is it accurate just to say, "To run Zowe Application Framework, installation of the Zowe Cross Memory Server (ZIS) is required." But isn't ZSS required to run certain API ML components and therefore, by extension, ZIS is also required?